pkg/notify/email.go

```diff
@@ -0,0 +1,40 @@
+package notify
+
+import (
+	"fmt"
+	"net/smtp"
+)
+
+const (
+	smtpServerAddress = "smtp.gmail.com"
+	smtpServerPort    = "587"
+)
+
+type Email struct {
+	Id   string
```

Comment:
```suggestion
	ID   string
```
just a Go way of naming variables :)